Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.yml

```diff
@@ -118,177 +127,344 @@ script:
-      description: Alert description
-      type: string
-    - contextPath: HelloWorld.Alert.status
-      description: Alert status, can be "open", "closed"
-      type: string
+      description: Alert name.
+      type: String
     - contextPath: HelloWorld.Alert.severity
-      description: Severity. 1-low,2-medium,3-high,4-critical
-      type: number
-    - contextPath: HelloWorld.Alert.type
-      description: 'Alert type. '
-      type: string
+      description: Alert severity.
+      type: String
   - arguments:
     - default: false
-      description: alert id
+      description: Alert ID.
       isArray: false
       name: alert_id
       required: true
       secret: false
     deprecated: false
-    description: Retrieve alert by id
+    description: Retrieve alert extra data by ID.
     execution: false
     name: helloworld-get-alert
     outputs:
-    - contextPath: HelloWorld.Alert.id
-      description: Alert id
-      type: string
-    - contextPath: HelloWorld.Alert.name
-      description: Alert name
-      type: string
+    - contextPath: HelloWorld.Alert.alert_id
+      description: Alert ID.
+      type: String
+    - contextPath: HelloWorld.Alert.created
+      description: Alert created time.
+      type: Number
     - contextPath: HelloWorld.Alert.description
-      description: Alert description
-      type: string
-    - contextPath: HelloWorld.Alert.status
-      description: Alert status, can be "open", "closed"
-      type: string
-    - contextPath: HelloWorld.Alert.severity
-      description: Severity. 1-low,2-medium,3-high,4-critical
-      type: number
-    - contextPath: HelloWorld.Alert.type
-      description: Alert type.
-      type: string
+      description: Alert description.
+      type: String
+    - contextPath: HelloWorld.Alert.device_id
+      description: ID of the device involved in the alert.
+      type: String
+    - contextPath: HelloWorld.Alert.device_ip
+      description: IP Address of the device involved in the alert.
+      type: String
+    - contextPath: HelloWorld.Alert.location
+      description: Location of the device involved in the alert.
+      type: String
+    - contextPath: HelloWorld.Alert.user
+      description: User involved in the alert.
+      type: String
+  - arguments:
+    - default: false
+      description: Alert ID to update.
+      isArray: false
+      name: alert_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: New status of the alert.
+      isArray: false
+      name: status
+      predefined:
+      - ACTIVE
+      - CLOSED
+      required: true
+      secret: false
+    deprecated: false
+    description: Update the status for an alert.
+    execution: false
+    name: helloworld-update-alert-status
+    outputs:
+    - contextPath: HelloWorld.Alert.alert_id
+      description: Alert ID.
+      type: String
+    - contextPath: HelloWorld.Alert.updated
+      description: Alert update time.
```

Comment:
format? why the update time is number?

Reply:
It's seconds since epoch, like the "created". Added a note to mention it. Should we convert it in Demisto before writing it in the context?

Reply:
Lets do it in the next time